src/tests/unit/studentFormData.test.js

```diff
@@ -0,0 +1,43 @@
+
+const filterFormData = (testData, cohortId) => testData.filter(app => app.id === cohortId)
+
+const cohortId = 'AKd12ds3'
+const testData = [
+    {
+        id: 'AKd123',
+        name: 'cohort 9',
+        type: 'development'
+    },
+    {
+      id: 'AKd12ds3',
+      name: 'cohort 8',
+      type: 'design'
+    },
+    {
+        id: 'AKd123ss',
+        name: 'cohort 10',
+        type: 'development'
+    }
+];
+
+describe("Filters for Correct Cohort Data", () => {
```

Comment:
If I were testing this function I would probably have one test to make sure it returns the right object (ie. your second test) and I would test that if I pass in a cohort id that isn't in the array it would return an empty array. There is absolutely nothing wrong with the other tests you wrote! They're just extra expects and you could probably have them all within one 'it' block. Each 'it' should be a different test case with likely some sort of change in the data

Reply:
that makes sense. Thank you, I'll implement that now